examples/framework/preferences/minimalEditor/schemas/primary.js

```diff
@@ -0,0 +1,37 @@
+ /*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+/* global fluid */
+
+(function () {
+    "use strict";
+
+   /**
+     * Primary Schema
+     * This schema defines the preference(s) edited by this preferences editor:
+     * their names, types, default values, etc.
+     */
+    fluid.defaults("minEditor.primarySchema", {
```

Comment:
The naming convention we used so far was to include the name of the preference in the primary schema grade. Something like this "minEditor.primarySchema.autoPilot". Although what we have here is technically valid, it may not be as scalable for adding new preferences to the editor. 

see: https://github.com/fluid-project/infusion/blob/master/src/framework/preferences/js/StarterSchemas.js#L165-L248

If you change this, the prefs framework tutorial will also need to be updated.